Makefile

```diff
@@ -0,0 +1,118 @@
+#   Copyright The beehive Authors.
+
+
+# Root directory of the project (absolute path).
+ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+# Used to populate variables in version package.
+VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
+REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
+GOVERSION=$(shell go version |awk -F ' ' '{printf $$3}')
+CURRENT_BRANCH=$(shell git symbolic-ref --short -q HEAD)
+
+ifneq "$(strip $(shell command -v go 2>/dev/null))" ""
+	GOOS ?= $(shell go env GOOS)
+	GOARCH ?= $(shell go env GOARCH)
+else
+	ifeq ($(GOOS),)
+		# approximate GOOS for the platform if we don't have Go and GOOS isn't
+		# set. We leave GOARCH unset, so that may need to be fixed.
+		ifeq ($(OS),Windows_NT)
+			GOOS = windows
+		else
+			UNAME_S := $(shell uname -s)
+			ifeq ($(UNAME_S),Linux)
+				GOOS = linux
+			endif
+			ifeq ($(UNAME_S),Darwin)
+				GOOS = darwin
+			endif
+			ifeq ($(UNAME_S),FreeBSD)
+				GOOS = freebsd
+			endif
+		endif
+	else
+		GOOS ?= $$GOOS
+		GOARCH ?= $$GOARCH
+	endif
+endif
+
+
+WHALE = "🇧 "
+OMG = "😱"
+
+PKG=github.com/kubeedge/beehive
+
+# Project packages.
+PACKAGES=$(shell go list ./... | grep -v "/vendor/")
+
+INTEGRATION_PACKAGE=${PKG}
+
+
+BUILDTAGS ?= 
+
+GO_TAGS=$(if $(BUILDTAGS),-tags "$(BUILDTAGS)",)
+
+GO_LDFLAGS=-ldflags '-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG) -X $(PKG)/version.GoVersion=$(GOVERSION) -X $(PKG)/version.Branch=$(CURRENT_BRANCH) $(EXTRA_LDFLAGS)'
+
+
+#Replaces ":" (*nix), ";" (windows) with newline for easy parsing
+GOPATHS=$(shell echo ${GOPATH} | tr ":" "\n" | tr ";" "\n")
+
+TESTFLAGS_RACE= -race
```

Comment:
you are using this variable only in line 76, please add the value on line 76 and remove this variable

Reply:
ok